pkg/asset/internal/templates.go

```diff
@@ -560,9 +600,12 @@ spec:
         resources:
           requests:
             cpu: 150m
-            memory: 10Mi
-      - name: dnsmasq-metrics
-        image: gcr.io/google_containers/dnsmasq-metrics-amd64:1.0
+            memory: 20Mi
+        volumeMounts:
+        - name: kube-dns-config
+          mountPath: /etc/k8s/dns/dnsmasq-nanny
```

Comment:
We don't create this directory (but we do create /etc/kubernetes). What is the dnsmasq nanny process / is this needed?